tests/foreman/cli/test_host.py

```diff
@@ -2012,22 +2012,31 @@ def test_negative_without_attach(
     assert "This system has no repositories available through subscriptions." in repo_list.stdout
 
 
+@pytest.mark.parametrize('sca_enabled', [True, False], ids=['with_sca', 'without_sca'])
 @pytest.mark.cli_host_subscription
 @pytest.mark.tier3
 def test_negative_without_attach_with_lce(
-    target_sat, host_subscription_client, function_org, function_lce
+    target_sat,
+    host_subscription_client,
+    function_org,
+    function_lce,
+    sca_enabled,
 ):
     """Attempt to enable a repository of a subscription that was not
-    attached to a host
+    attached to a host.
     This test is not using the host_subscription entities except
     subscription_name and repository_id
 
     :id: fc469e70-a7cb-4fca-b0ea-3c9e3dfff849
 
-    :expectedresults: repository not enabled on host
+    :expectedresults: Repository enabled with SCA. Failure without SCA.
 
     :parametrized: yes
     """
+    if sca_enabled:
+        function_org.sca_enable()
+    else:
+        function_org.sca_disable()
```

Comment:
As SCA will be enabled as default here, so we won't need to enable here 
```suggestion
    if not sca_enabled:
        function_org.sca_disable()
```

Reply:
I'd prefer to keep this as it is, in case it is not enabled by default (perhaps in future z-stream).